ArcTouchChallenge/ArcTouchChallenge/Model/Movie/Movie.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Results.swift
+//
+//  Created by Arthur Augusto Sousa Marques on 3/13/18
+//  Copyright (c) . All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Movie: Model {
+
+  // MARK: Declaration for string constants to be used to decode and also serialize.
+  private struct SerializationKeys {
+    static let posterPath = "poster_path"
+    static let backdropPath = "backdrop_path"
+    static let genreIds = "genre_ids"
+    static let voteCount = "vote_count"
+    static let overview = "overview"
+    static let originalTitle = "original_title"
+    static let voteAverage = "vote_average"
+    static let popularity = "popularity"
+    static let id = "id"
+    static let originalLanguage = "original_language"
+    static let releaseDate = "release_date"
+    static let video = "video"
+    static let title = "title"
+    static let adult = "adult"
+  }
+
+  // MARK: Properties
+  public var posterPath: String?
+  public var backdropPath: String?
+  public var genreIds: [Int]?
+  public var voteCount: Int?
+  public var overview: String?
+  public var originalTitle: String?
+  public var voteAverage: Float?
+  public var popularity: Float?
+  public var id: Int?
+  public var originalLanguage: String?
+  public var releaseDate: String?
+  public var video: Bool? = false
+  public var title: String?
+  public var adult: Bool? = false
+
+  public var imageData: Data?
+
+  // MARK: SwiftyJSON Initializers
+  /// Initiates the instance based on the object.
+  ///
+  /// - parameter object: The object of either Dictionary or Array kind that was passed.
+  /// - returns: An initialized instance of the class.
+  public convenience init(object: Any) {
```

Comment:
This was created automatically from the component third party and wasn't necessary yet to be used, but it could be if there was a necessity to create a object hard coded.